src/Entity/User.php

```diff
@@ -45,6 +52,11 @@ class User extends AbstractBaseEntity implements UserInterface, PasswordAuthenti
      */
     private ?string $provider = null;
 
+    /**
+     * @ORM\Column(type="string", nullable=true, enumType="App\Enum\UserTypeEnum")
+     */
+    private ?UserTypeEnum $userType = null;
```

Comment:
Why is this nullable? Should all users not have a type?

Reply:
Fixed